alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/solr/sharding/SolrShardingMetricsFactory.java

```diff
@@ -26,6 +33,15 @@ public SolrShardingMetricsFactory(ShardRegistry shardRegistry, MeterRegistry reg
                 .forJob(jobDetail.getKey()).build();
 
         scheduler.scheduleJob(jobDetail, trigger);
+        this.trigger = trigger;
+    }
+
+    public void destroy() {
+        try {
+            scheduler.unscheduleJob(trigger.getKey());
+        } catch (SchedulerException e) {
+            LOGGER.trace("Unable to unschedule job in SolrShardingMetricsFactory", e);
```

Comment:
`TRACE` is the most fine-grained level (even more fine-grained than `DEBUG`)

I think you want to at least log on the level of `WARN`, possibly even `ERROR`.

There is no need to repeat the class name in the message. Depending on the formatter that is used the message will be something like `[ALFRESCO] 2020-11-18 14:50:23 UTC [e.x.a.t.b.solr.sharding.SolrShardingMetricsFactory] TRACE: Unable to unschedule job in SolrShardingMetricsFactory`